src/scripting/LuaSandbox.cpp

```diff
@@ -507,7 +507,8 @@ void LuaSandbox::InitializeIOForSandbox(Sandbox& aSandbox, const sol::state& acp
         for (const auto& entry : std::filesystem::directory_iterator(path))
         {
             sol::table item(stateView, sol::create);
-            item["name"] = UTF16ToUTF8(relative(entry.path(), path).native());
+            item["relativePath"] = UTF16ToUTF8(Relative(entry.path(), path).native());
+            item["name"] = UTF16ToUTF8(entry.path().filename().native());
```

Comment:
this returned relative path formerly, seems a bit sus...

Reply:
Yeah...